docs/solidity-by-example.rst

```diff
@@ -645,99 +645,71 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-In this section we will learn how to build a simple implementation
-of a payment channel. It use cryptographics signatures to make
+In this section we will learn how to build an example implementation
+of a payment channel. It use cryptographic signatures to make
 repeated transfers of Ether between the same parties secure, instantaneous, and
-without transaction fees. To do it we need to understand how to
+without transaction fees. For the example, we need to understand how to
 sign and verify signatures, and setup the payment channel.
 
 Creating and verifying signatures
 =================================
 
 Imagine Alice wants to send a quantity of Ether to Bob, i.e.
 Alice is the sender and the Bob is the recipient.
+.. TODO: What does the below mean?
 Alice only needs to send cryptographically signed messages off-chain
 (e.g. via email) to Bob and it will be very similar to writing checks.
 
-Signatures are used to authorize transactions,
-and they are a general tool that is available to
-smart contracts. Alice will build a simple
-smart contract that lets her transmit Ether, but
-in a unusual way, instead of calling a function herself
-to initiate a payment, she will let Bob
-do that, and therefore pay the transaction fee.
+Alice and Bob use signatures to authorise transactions, which are a feature of smart contracts.
+Alice will build a simple smart contract that lets her transmit Ether, but instead of calling a function herself
+to initiate a payment, she will let Bob do that, and therefore pay the transaction fee.
+
 The contract will work as follows:
 
     1. Alice deploys the ``ReceiverPays`` contract, attaching enough Ether to cover the payments that will be made.
-    2. Alice authorizes a payment by signing a message with their private key.
+    2. Alice authorises a payment by signing a message with their private key.
     3. Alice sends the cryptographically signed message to Bob. The message does not need to be kept secret
-       (you will understand it later), and the mechanism for sending it does not matter.
+       (explained later), and the mechanism for sending it does not matter.
     4. Bob claims their payment by presenting the signed message to the smart contract, it verifies the
        authenticity of the message and then releases the funds.
 
 Creating the signature
 ----------------------
 
-Alice does not need to interact with Ethereum network to
-sign the transaction, the proccess is completely offline.
-In this tutorial, we will sign messages in the browser
-using ``web3.js`` and ``MetaMask``.
-In particular, we will use the standard way described in `EIP-762 <https://github.com/ethereum/EIPs/pull/712>`_,
+Alice does not need to interact with the Ethereum network to sign the transaction, the process is completely offline.
```

Comment:
Please try to change the newlines as little as possible (at least in such a large PR) as it makes the review much more complicated.

Reply:
@chriseth I always try, but when you rewrite and reorder entire sentences it's not always possible, and kind of makes looking and writing text challenging. When I review text I tend to review the whole thing, not diffs, as it's not quite the same as code.

I can reset all line breaks to a certain character number on every PR to keep it consistent, but no guarantee others will do that, and I don't know if that solves it anyway. Maybe this is something to discuss at some point.

Also in this PR submission there were lots of extraneous line breaks to remove.